.doc_gen/metadata/sesv2_metadata.yaml

```diff
@@ -110,5 +158,208 @@ sesv2_SendEmail:
+              snippet_tags:
+                - sesv2.rust.create-email-template
+  services:
+    sesv2: {CreateEmailTemplate}
+
+sesv2_DeleteContactList:
+  title: Delete an &SESv2; contact list using an &AWS; SDK
+  title_abbrev: Delete a  contact list
+  synopsis: delete an &SESv2; contact list.
+  category:
+  languages:
+    Java:
+      versions:
+        - sdk_version: 2
+          github: javav2/example_code/ses
+          excerpts:
+            - description:
+              snippet_tags:
+                - sesv2.java2.newsletter.DeleteContactList
+    Python:
+      versions:
+        - sdk_version: 3
+          github: python/example_code/sesv2
+          excerpts:
+            - description:
+              snippet_tags:
+                - python.example_code.sesv2.DeleteContactList
+    Rust:
+      versions:
+        - sdk_version: 1
+          github: rustv1/examples/ses
+          excerpts:
+            - description:
+              snippet_tags:
+                - sesv2.rust.delete-contact-list
+  services:
+    sesv2: {DeleteContactList}
+
+sesv2_DeleteEmailIdentity:
+  title: Delete an &SESv2; email identity using an &AWS; SDK
+  title_abbrev: Delete an email identity
+  synopsis: delete an &SESv2; email identity.
+  category:
+  languages:
+    Java:
+      versions:
+        - sdk_version: 2
+          github: javav2/example_code/ses
+          excerpts:
+            - description:
+              snippet_tags:
+                - sesv2.java2.newsletter.DeleteEmailIdentity
+    Python:
+      versions:
+        - sdk_version: 3
+          github: python/example_code/sesv2
+          excerpts:
+            - description:
+              snippet_tags:
+                - python.example_code.sesv2.DeleteEmailIdentity
+    Rust:
+      versions:
+        - sdk_version: 1
+          github: rustv1/examples/ses
+          excerpts:
+            - description:
+              snippet_tags:
+                - sesv2.rust.delete-email-identity
+  services:
+    sesv2: {DeleteEmailIdentity}
+
+sesv2_DeleteEmailTemplate:
+  title: Delete an &SESv2; email template using an &AWS; SDK
+  title_abbrev: Delete an email template
+  synopsis: delete an &SESv2; email template.
+  category:
+  languages:
+    Java:
+      versions:
+        - sdk_version: 2
+          github: javav2/example_code/ses
+          excerpts:
+            - description:
+              snippet_tags:
+                - sesv2.java2.newsletter.DeleteEmailTemplate
+    Python:
+      versions:
+        - sdk_version: 3
+          github: python/example_code/sesv2
+          excerpts:
+            - description:
+              snippet_tags:
+                - python.example_code.sesv2.DeleteEmailTemplate
+    Rust:
+      versions:
+        - sdk_version: 1
+          github: rustv1/examples/ses
+          excerpts:
+            - description:
+              snippet_tags:
+                - sesv2.rust.delete-email-template
+  services:
+    sesv2: {DeleteEmailTemplate}
```

Comment:
Where is the metadata for the scenario? I don't see the workflow listed as a scenario in the build, either.

Reply:
It's missing `category: Scenarios` so it still isn't showing up as a scenario.

Reply:
Done